modules/imgproc/test/test_color.cpp

```diff
@@ -3072,20 +3072,40 @@ TEST(ImgProc_RGB2YUV, regression_13668)
     EXPECT_EQ(res, ref);
 }
 
-TEST(ImgProc_cvtColorTwoPlane, missing_check_17036)  // test can be removed if required feature is implemented
+TEST(ImgProc_cvtColorTwoPlane, regression_17036)
 {
-    std::vector<uchar> y_data(700 * 480);
-    std::vector<uchar> uv_data(640 * 240);
+    RNG rng;
 
-    Mat y_plane_padding(480, 640, CV_8UC1, y_data.data(), 700);  // with stride
-    Mat uv_plane(240, 320, CV_8UC2, uv_data.data());
+    std::vector<uchar> y(640 * 480,  (uchar)rng.uniform(16, 235));
+    Mat y_mat(480, 640, CV_8UC1, y.data());
```

Comment:
`RNG::fill` can fill Matrix directly. No need to create extra container.
https://docs.opencv.org/master/d1/dd6/classcv_1_1RNG.html#ad26f2b09d9868cf108e84c9814aa682d

Reply:
Fixed.